whylog_vim/output_formater/input_windows_messages.py

```diff
@@ -58,17 +121,41 @@ def get_primary_key_message(cls, parser):
         output.add_commented(Messages.PRIMARY_KEY)
         cls._add_parser(output, parser)
         output.add_commented(Messages.ENDING)
+        output.add(', '.join([str(num) for num in parser.primary_key_groups]))
```

Comment:
allocating a list here is not necessary, so a generator expression is sufficient:
```python
output.add(', '.join(str(num) for num in parser.primary_key_groups))
```